.eslintrc.yml

```diff
@@ -1,15 +1,18 @@
 env:
   es6: true
   node: true
-parser: babel-eslint
-extends:
-- eslint:recommended
-- plugin:flowtype/recommended
+  jest: true
 plugins:
-- flowtype
+  - flowtype
+  - jest
+extends:
```

Comment:
Soon, https://github.com/verdaccio/core/issues/3 in case you want to use it. We will migrate all plugins to such dependencies.